tracee/tracee.go

```diff
@@ -341,6 +341,50 @@ func (t *Tracee) processEvent(ctx *context, args []interface{}) error {
 		}
 	}
 
+	//capture executed files
+	if t.config.CaptureFiles && (eventName == "security_bprm_check") {
+		var err error
+
+		destinationDirPath := filepath.Join(t.config.OutputPath, strconv.Itoa(int(ctx.Mnt_id)))
+		if err := os.MkdirAll(destinationDirPath, 0755); err != nil {
+			return err
+		}
+		sourceFilePath, ok := args[0].(string)
+		if !ok {
+			return fmt.Errorf("error parsing security_bprm_check args")
+		}
+		destinationFilePath := filepath.Join(destinationDirPath, fmt.Sprintf("exec.%d.%s", ctx.Ts, filepath.Base(sourceFilePath)))
```

Comment:
I'm not sure we need the file path in the new filename. It is possible to extract the filename by looking at the event output. What do you think?

Reply:
it's not the file path, it's just the file name

Reply:
to explain why I chose this name: I wanted to include the file name, for example `ls`. but that is not unique for example if someone execed ls and then renamed another file to be called "ls", so I added the timestamp. so the true identifier is the timestamp, the filename is added for convenience